src/rqt_console/message_proxy_model.py

```diff
@@ -30,21 +28,17 @@
 # ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
-from python_qt_binding.QtCore import Qt, qVersion, qWarning
-try:
-    from python_qt_binding.QtCore import QSortFilterProxyModel  # Qt 5
-except ImportError:
-    from python_qt_binding.QtGui import QSortFilterProxyModel  # Qt 4
-from python_qt_binding.QtGui import QBrush, QColor
+from python_qt_binding.QtCore import QSortFilterProxyModel
+from python_qt_binding.QtCore import Qt
```

Comment:
If Qt 4 support is removed you can combine these lines.
```suggestion
from python_qt_binding.QtCore import QSortFilterProxyModel, Qt
```